packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue

```diff
@@ -1,12 +1,22 @@
 <template>
 	<div class="p-datatable-wrapper">
-		<Dropdown
-			v-if="matrixMap && Object.keys(matrixMap).length > 0"
-			:model-value="matrixType"
-			:options="matrixTypes"
-			placeholder="Select matrix type"
-			@update:model-value="(v) => changeMatrix(v)"
-		/>
+		<div class="matrix-toolbar">
+			<Dropdown
+				v-if="matrixMap && Object.keys(matrixMap).length > 0"
+				:model-value="matrixType"
+				:options="matrixTypes"
+				placeholder="Select matrix type"
+				@update:model-value="(v) => changeMatrix(v)"
+			/>
+
+			<Button
```

Comment:
Very minor but it is hard to distinguish the difference of enabled vs disabled
Changing this to `severity="primary"` would fix this. (green vs not green)
Otherwise I can ask design channel about changing the two colours for this severity in a separate PR

<img width="319" alt="image" src="https://github.com/user-attachments/assets/60371a9b-2ad7-4e91-b7a8-68ad634cef88">

<img width="322" alt="image" src="https://github.com/user-attachments/assets/c6611e3a-d58e-4fd2-bba3-7bd013f8539e">

Reply:
I was about to say the same thing, was confused about the button state ^

Reply:
Let's get this out first so people and poke holes, and we can have a tweaks/UX round.